manifests/apply.sh

```diff
@@ -0,0 +1,6 @@
+oc apply -f hive_v1alpha1_syncset.yaml 
```

Comment:
Is this a testing script as well?

Reply:
Yes this is temporary for testing purposes.

Reply:
Removed testing files.